index.html

```diff
@@ -25,5 +25,99 @@ <h3 class="text2">I'm a Software Developer</h3>
     </ul>
     <div class="illustration"></div>
     </section>
+
+  <section class="work"><h2>My Recent Works</h2>
+    <div class="vector"><img src="icons/Vector.png"></div>
+    <div class="image1"><img src="icons/image1.png"></div> 
+    <div class="texts1">Multi-Post Stories</div>
+    <div><p class="texts2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p></div>
+    <div class="tags"><img src="icons/Tags.png"></div>
+    <div class="see"><img src="icons/see-project.png"></div>
+    <div class="grid-container">
+        <div class="first">
+          <h4>Professional Art Printing Data</h4>
+          <p class="ntext">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <nav class="button">
+           <p class="html"><img src="icons/htmll.png"></p>
+           <p class="boot"><img src="icons/bootstrap.png"></p>
+           <p class="ruby"><img src="icons/ruby.png"></p>
+           </nav>
+           <div class="see2"><img src="icons/see2.png"></div>
+        </div>
+        <div class="second">
+          <h4>Professional Art Printing Data</h4>
+          <p class="ntext">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <nav>
+           <p class="html"><img src="icons/htmll.png"></p>
+           <p class="boot"><img src="icons/bootstrap.png"></p>
+           <p class="ruby"><img src="icons/ruby.png"></p>
+        </nav>
+           <div class="see2"><img src="icons/see2.png"></div>
+        </div>
+        <div class="third">
+          <h4>Professional Art Printing Data</h4>
+          <p class="ntext">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <nav class="button">
+           <p class="html"><img src="icons/htmll.png"></p>
+           <p class="boot"><img src="icons/bootstrap.png"></p>
+           <p class="ruby"><img src="icons/ruby.png"></p>
+           </nav>
+           <div class="see2"><img src="icons/see2.png"></div>
+        </div>
+        <div class="fourth">
+          <h4>Professional Art Printing Data</h4>
+          <p class="ntext">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <nav class="button">
+           <p class="html"><img src="icons/htmll.png"></p>
+           <p class="boot"><img src="icons/bootstrap.png"></p>
+           <p class="ruby"><img src="icons/ruby.png"></p>
+           </nav>
```

Comment:
This is a list of items (project technologies) so kindly use `ul` and `li` tags to list them. I see that you created them as images, but it  should just use the list tags and a border around them. You can consider doing something similar below :point_down:  and use css to add the border around it. This applies to all the other project cards.

              <ul>
                    <li>HTML</li>
                    <li>Bootstrap</li>
                    <li>Ruby</li>
                </ul>